modules/private-service-connect/outputs.tf

```diff
@@ -0,0 +1,63 @@
+/**
+ * Copyright 2022 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+output "private_service_connect_name" {
+  value       = var.private_service_connect_name
+  description = "Private service connect name"
+
+  depends_on = [
+    google_compute_global_forwarding_rule.forwarding_rule_private_service_connect
+  ]
+}
+
+output "private_service_connect_ip" {
+  value       = var.private_service_connect_ip
```

Comment:
Lets not plumb user provided vars back as outputs.

Reply:
Few instances of these here, lets remove those.

Reply:
I retargeted this values to actual created resources